code/modules/client/preference/loadout/loadout_hat.dm

```diff
@@ -113,32 +108,54 @@
 	display_name = "cowboy hat, pink"
 	path = /obj/item/clothing/head/cowboyhat/pink
 
-/datum/gear/hat/pr_beret
+//Berets Refactor
+//Defining all Job Berets as no-overlap
+//Somehow this doesn't break the DB.
```

Comment:
For some unknown reason, we store loadout items as a URL string in the DB.

`mantle%2c+head+of+personnel=&military+jacket%2c+desert=&leather+jacket=`

I hate it, but it makes this PR easier atleast